patitasuy/src/app/pages/profile/profile.component.html

```diff
@@ -1,9 +1,63 @@
-<ion-header></ion-header>
+<ion-header class="bar">
+  <mat-icon class="container-iconPets">pets</mat-icon>
+  <ion-title class="title">Perfil</ion-title>
+</ion-header>
+
 <ion-content>
-  <p>
-    profile works!
-  </p>
+  <div class="userContent">
+    <ion-icon class="container-iconAccount" name="person"></ion-icon>
+    <div class="userDates">
+      <p class="userName">{{name}}</p>
+      <p class="userEmail">{{email}}</p>
+    </div>
+    <ion-button id="open-modal" class="editButton" expand="block">
+      <ion-icon name="create-outline"></ion-icon>
+    </ion-button>
+    <ion-modal trigger="open-modal">
+      <ng-template>
+        <ion-header>
+          <ion-toolbar>
+            <ion-buttons slot="start">
+              <ion-button (click)="cancel()">Cancel</ion-button>
+            </ion-buttons>
+            <ion-buttons slot="end">
+              <ion-button (click)="confirm()" [strong]="true">Confirm</ion-button>
+            </ion-buttons>
+          </ion-toolbar>
+        </ion-header>
+        <ion-content class="ion-padding">
+          <ion-item>
+            <ion-label position="stacked">Ingresa tu nombre</ion-label>
+            <ion-input type="text" placeholder="Nombre" [(ngModel)]="name"></ion-input>
+          </ion-item>
+          <ion-item>
+            <ion-label position="stacked">Ingresa tu email</ion-label>
+            <ion-input type="text" placeholder="Email" [(ngModel)]="email"></ion-input>
+          </ion-item>
+        </ion-content>
+      </ng-template>
+    </ion-modal>
+  </div>
+  <ion-segment (ionChange)="selectPublications($event)">
```

Comment:
Add the attribute "value" with the value "mis-publicaciones" so as to leave the mis-publicaciones tab as selected